supported-version/src/versions/magento-open-source/individual.json

```diff
@@ -669,6 +697,20 @@
         "nginx": "nginx:1.22",
         "os": "ubuntu-latest",
         "release": "2025-02-11T00:00:00+0000",
+        "eol": "2025-04-07T00:00:00+0000"
+    },
+    "magento/project-community-edition:2.4.6-p10": {
+        "magento": "magento/project-community-edition:2.4.6-p10",
+        "php": 8.1,
```

Comment:
```suggestion
        "php": 8.2,
```

Reply:
@damienwebdev is 2.4.6-p10 really changing the php version requirement?

Reply:
checking https://experienceleague.adobe.com/en/docs/commerce-operations/installation-guide/system-requirements 2.4.6-p10 still supports 8.1